CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # App Center SDK for Android Change Log
 
-## Version 2.3.1 (Under development)
+## Version 2.4.0 (Under development)
+
+### App Center Auth
+
+* **[Feature]** Add authentication support for Azure Active Directory (AAD) applications. Users can now connect an [AAD tenant and AAD single tenant (AzureADMyOrg), multi-tenant (AzureADMultipleOrgs) and multi-tenant and personal Microsoft account (AzureADandPersonalMicrosoftAccount)](https://docs.microsoft.com/en-us/azure/active-directory/develop/single-and-multi-tenant-apps) applications.
```

Comment:
The link provided: https://docs.microsoft.com/en-us/azure/active-directory/develop/single-and-multi-tenant-apps, doesn't really match the terms used in the descriptions, and as a user, I still have questions such as "What is Azure ADMyOrg", "AzureADMultipleOrgs", "AzureADandPersonalMicrosoftAccount"? 

A suggestion would be to either 
1. Find a documentation link that directly explains what the terms "AzureADMyOrg", "AzureADMultipleOrgs", "AzureADandPersonalMicrosoftAccount" mean. Is there such a doc? If so, let's link it.
2. Directly explain the user scenarios from the [linked doc](https://docs.microsoft.com/en-us/azure/active-directory/develop/single-and-multi-tenant-apps). See below for an example of how it would look like. We don't have to use this doc, if there is better documentation that you'd recommend, let's use that.

```suggestion
* **[Feature]** Add authentication support for Azure Active Directory (AAD) applications. Users can now connect an AAD tenant with [AAD single-tenant or multi-tenant apps](https://docs.microsoft.com/azure/active-directory/develop/single-and-multi-tenant-apps) with the following audiences:
  - Accounts in this directory only
  - Accounts in any Azure AD directory
  - Accounts in any Azure AD directory and personal Microsoft accounts
```

fyi @manuchandel